### DISCUSSION TITLE
MOTD display

### DISCUSSION THREAD

**Discussion by Zathiel** [gravitational/teleport#46359] [Q&A]
Expected behavior:

I wanted to see my MOTD as soon as someone log from web UI or CLI

Current behavior:

MOTD not displayed 

Bug details:
- Teleport version : 16.2.x
- Recreation steps : Login to Telegram UI /tzh login using admin credentials nothing displayed on my ressources and even if i log on on the followed ressources.

- Debug logs

Not yet export

**Comment by zmb3** [comment]
The MOTD is displayed in the web UI prior to logging in, and in the CLI after running `tsh login`.

If you find that not to be the case, please add detailed reproduction steps to this issue.

**Comment by Zathiel** [comment]
hey @zmb3 

thanks for your fast reply.

Initial messages edited im not sure what happnd 

Regards

**Comment by zmb3** [comment]
I'm going to convert this to a discussion, as it's not actionable until you provide detailed reproduction steps. (Did you even configure a MOTD? It's hard to say without seeing your configuration)

Additionally, Teleport 16.4.0 has not been released.

**Comment by Zathiel** [comment]
/etc/motd has been set up  

I confirm 16.2.x my apologies

When i login using putty my MOTD is displaying well but on teleport nothing

@zmb3

**Reply by zmb3** [reply]
https://goteleport.com/docs/enroll-resources/server-access/guides/ssh-pam/#display-a-message-of-the-day-motd-with-teleport

**Reply by stevenGravy** [reply]
> /etc/motd has been set up
> 
> I confirm 16.2.x my apologies
> 
> When i login using putty my MOTD is displaying well but on teleport nothing
> 
> @zmb3

If you want the ssh server MOTD to display then you need to enable PAM on the Teleport config. For the servers you are running the Teleport SSH service add the `pam` part as below. 

```yaml
ssh_service:
  enabled: true
  pam:
    enabled: true
```
